lib/hooks/useFormTimer.ts

```diff
@@ -30,7 +30,9 @@ export const useFormTimer = (): [FormTimerStateType, FormTimerDispatchType] => {
   ): { canSubmit: boolean; remainingTime: number; timerDelay?: number; timeLock?: number } {
     switch (action.type) {
       case "START": {
-        if (!action.timerDelay) throw new Error("Missing timerDelay on start FormTimer action");
```

Comment:
This should be left as is, since 0 is intentionally the disabled state. So don't use 0 as the default